client/src/lib/solana/transaction.ts

```diff
@@ -38,4 +45,45 @@ export class SolidTransaction {
     const recordKey = await getKeyFromAuthority(authority);
     return SolidTransaction.getSolid(connection, recordKey);
   }
+
+  static async deactivateSolid(
+    connection: Connection,
+    payer: Account,
+    recordKey: PublicKey
+  ): Promise<string> {
+    // Create the transaction to close the Solid DID account
+    // The payer must have permissions to deactivate the DID
+    // The payer receives the lamports stored in the DID account
+    const transaction = new Transaction().add(
+      closeAccount(recordKey, payer.publicKey, payer.publicKey)
+    );
+
+    // Send the instructions
+    return SolanaUtil.sendAndConfirmTransaction(connection, transaction, payer);
+  }
+
+  static async updateSolid(
+    connection: Connection,
+    payer: Account,
```

Comment:
We can also add the authority separately for this instruction

Reply:
Agreed. I was thinking along the lines of keeping the API simple but you are probably right